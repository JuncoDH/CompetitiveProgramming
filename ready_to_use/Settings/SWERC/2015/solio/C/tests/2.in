100
20
22 28 18 3 14 16 27 6 1 16 8 1 2 8 13 30 3 27 4 1
1
8
1
4
8
26 19 4 29 28 5 25 18
13
1 9 9 15 21 17 26 10 17 22 17 9 12
8
7 30 19 13 20 17 9 24
4
7 28 8 12
16
19 6 29 28 2 21 12 23 14 14 11 4 6 24 6 9
15
1 7 27 10 26 10 24 1 29 5 8 17 28 26 18
16
30 6 24 13 26 11 28 29 25 9 13 14 22 4 1 12
12
9 22 6 25 7 11 29 3 1 20 8 12
2
4 1
7
30 3 14 21 8 14 26
15
24 17 2 9 21 26 22 27 22 16 24 12 11 25 26
12
29 8 8 6 23 29 15 28 13 24 15 15
8
27 27 1 8 20 14 10 1
16
28 29 15 24 6 12 8 13 30 30 11 10 17 24 3 10
17
30 22 12 16 6 15 11 26 21 19 9 16 7 2 1 29 1
14
4 23 23 5 20 24 17 23 1 28 28 30 28 24
5
23 26 25 19 27
8
20 24 5 13 12 28 15 22
19
13 10 12 7 4 3 11 14 11 22 8 23 9 6 12 20 21 20 29
12
23 29 3 20 29 10 13 19 4 29 13 29
7
24 25 15 18 1 5 24
3
18 4 14
5
14 13 14 25 26
20
25 9 13 21 19 26 10 2 5 14 23 16 1 14 29 9 19 26 25 13
6
29 13 15 1 23 26
17
22 26 4 18 15 22 21 15 23 7 26 15 13 1 18 15 10
17
12 30 7 15 17 11 8 4 18 19 20 23 20 12 22 13 9
18
26 9 5 21 14 23 11 11 15 22 15 21 13 3 2 25 2 16
20
20 4 16 12 26 28 27 9 27 5 5 30 8 29 18 11 25 16 29 7
4
28 4 17 4
17
10 22 12 4 30 5 6 1 20 13 29 11 27 15 4 25 28
4
24 25 20 2
13
30 23 26 3 9 4 17 30 18 20 8 21 27
15
1 23 8 3 24 2 17 23 1 28 9 30 23 8 26
13
12 13 14 18 13 22 2 1 8 22 15 26 7
19
6 28 10 27 11 16 2 19 8 3 8 8 20 5 10 24 21 18 26
11
20 12 27 7 15 14 30 12 16 23 4
6
4 11 1 30 17 1
1
1
2
7 23
12
5 5 9 10 19 2 4 7 18 29 4 5
15
9 24 6 19 26 16 12 19 12 9 2 7 26 22 22
20
9 28 5 3 3 18 8 12 7 9 12 28 4 11 17 13 15 20 14 23
6
27 17 25 7 27 18
12
27 17 3 1 20 18 18 17 9 30 28 9
4
25 29 21 12
14
24 15 2 4 8 7 30 26 8 16 12 3 14 16
6
26 16 21 8 18 8
4
20 1 10 30
1
6
10
21 27 22 16 14 19 12 27 30 30
7
16 24 29 12 9 5 11
14
22 13 11 3 20 21 3 2 18 24 23 3 10 20
16
20 2 21 5 3 21 21 3 10 11 30 28 17 28 6 6
19
14 5 25 21 1 19 13 24 20 20 12 27 22 19 12 26 18 20 19
9
15 24 12 30 21 30 20 24 1
17
19 5 26 5 21 4 18 16 17 8 17 17 23 7 27 24 5
18
5 13 10 28 23 22 25 13 10 10 24 14 1 19 5 26 22 12
18
13 22 11 27 17 4 19 28 6 15 5 21 18 25 3 9 30 19
19
2 10 18 9 23 1 22 5 25 20 3 5 3 16 12 28 1 11 17
16
23 24 28 13 25 1 16 27 23 12 3 25 5 21 20 9
15
20 15 23 5 28 28 8 12 25 20 26 21 2 26 9
6
5 7 12 7 1 29
8
21 20 2 29 11 30 5 27
17
8 5 3 8 9 8 8 10 16 7 29 11 13 19 9 22 25
18
24 26 28 1 1 2 15 26 29 3 2 12 22 10 12 23 9 11
15
8 21 15 28 27 7 23 24 4 1 24 20 28 13 27
16
13 20 24 11 25 25 3 2 9 20 30 14 23 9 26 2
12
5 6 18 10 28 18 3 9 15 10 29 12
7
17 22 26 22 16 9 26
9
15 22 4 13 7 20 29 12 26
1
4
10
30 6 13 16 7 3 11 21 30 25
16
22 10 24 18 15 12 4 27 10 2 23 24 20 2 13 30
7
20 22 22 22 5 12 1
12
17 24 6 11 15 6 19 18 3 11 30 19
10
14 12 30 7 16 7 29 8 23 8
9
28 3 15 29 7 10 20 3 15
3
21 8 18
8
28 3 14 10 10 7 10 2
7
30 23 26 10 5 19 27
18
25 9 30 4 16 21 18 22 28 5 5 11 25 24 20 22 30 13
12
11 20 16 7 25 15 22 15 27 4 22 1
13
27 30 3 19 3 22 23 1 27 1 24 30 21
6
4 20 7 20 19 11
14
14 12 6 28 16 17 7 26 2 7 1 3 25 18
5
14 2 1 3 16
4
16 21 25 19
18
28 26 8 19 4 4 25 1 10 16 3 1 19 18 18 15 18 7
19
6 1 19 15 22 9 4 8 5 21 20 20 8 2 3 27 3 25 28
2
22 16
13
18 3 17 13 1 25 20 15 12 5 15 6 19
16
30 25 16 6 5 26 5 14 10 23 24 16 13 15 10 12
11
10 5 27 13 23 9 7 7 20 16 22
10
24 20 15 20 11 26 20 10 29 30
9
27 26 14 23 19 21 24 14 26
2
4 28
