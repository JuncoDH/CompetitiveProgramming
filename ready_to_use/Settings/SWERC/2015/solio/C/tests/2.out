954
0
0
441
664
403
97
858
853
1014
501
5
303
1090
733
282
1032
918
1013
282
406
1044
749
290
54
211
1320
257
1071
1037
1008
1460
86
1000
139
830
795
602
1085
600
123
0
30
378
867
1066
308
664
174
657
242
103
0
715
279
684
859
1397
542
1012
1132
1166
897
1051
1007
131
401
818
974
1007
965
552
387
449
0
505
971
270
613
488
375
73
227
378
1322
625
778
195
633
65
162
924
958
38
590
965
527
671
610
32
